src/main/webapp/dev/js/common/feedbackResponseComments.js

```diff
@@ -169,8 +169,6 @@ function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
     }
     $(`#showResponseCommentAddForm${id}`).show();
 
-    $(`#responseCommentAddForm${id}`).empty();
-
     if (typeof richTextEditorBuilder !== 'undefined') {
         /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
         richTextEditorBuilder.initEditor(`#responseCommentAddForm${id}`, {
```

Comment:
So a possible solution is mentioned below, there is some changes in behavior but I doubt it will break anything 
 though we can't be sure without comprehensively tracing the usage. In any case, we do have tests and this is not going to cause a major problem if it breaks. Let's clean the code up.

```Javascript
const responseCommentAddFormId = `responseCommentAddForm${id}`;
const responseCommentEditor = tinymce.get(responseCommentAddFormId);
if (responseCommentEditor === null) {
    richTextEditorBuilder.initEditor(responseCommentAddFormId, {
        inline: true,
    });
} else {
    responseCommentEditor.setContent('');
}
```